lua/nvim-web-devicons.lua

```diff
@@ -1343,6 +1343,30 @@ local icons = {
     cterm_color = "67",
     name = "Solidity",
   },
+  [".env"] = {
+    icon = "",
+    color = "#faf743",
+    cterm_color = "226",
+    name = "Env",
+  },
+  ["prisma"] = {
+    icon = "卑",
+    color = "#ffffff",
+    cterm_color = "white",
+    name = "Prima",
+  },
+  ["lock"] = {
+    icon = "",
+    color = "#bbbbbb",
+    cterm_color = "250",
+    name = "Lock",
+  },
+  ["log"] = {
+    icon = "",
```

Comment:
not sure about this one, why is this a logout like icon ?

Reply:
I can't find an appropriate icon for this one. So i choose `login` icon with the meaning: "Write into a file"

Is there any suggestion?

Reply:
maybe `` or ``

Reply:
Thank you. I've changed it to the latter.